index.html

```diff
@@ -1122,37 +1122,91 @@
 
 ### Ancillary uses
 
-In order to uphold the principle of [[[#data-minimization]]], [=sites=] and
-[=user agents=] should seek to understand and respect people's goals and preferences about
-use of data about them.
-
 [=Sites=] sometimes use data in ways that aren't needed for the user's immediate
-goals. These uses are known as <dfn data-lt="ancillary use">ancillary uses</dfn>,
-and data that is primarily useful for [=ancillary uses=] is <dfn>ancillary data</dfn>.
+goals. For example, they might bill advertisers, measure site performance, or
+tell developers about bugs. These uses are known as <dfn data-lt="ancillary
+use">ancillary uses</dfn>.
 
-<aside class="example">
-  Some examples of [=ancillary data=] include data used for browser telemetry, site telemetry,
-  performance measurements, and software updates.
-</aside>
+[=Sites=] can get the data they want for [=ancillary uses=] from a variety of places:
+
+<dl>
+  <dt><dfn>Non-ancillary APIs</dfn></dt>
+  <dd>
+    Web APIs that were designed to support users' immediate goals, like <a
+    data-cite="dom#interface-event">DOM events</a> and <a
+    data-cite="cssom-view-1#extension-to-the-element-interface">element position
+    observers</a>.
+  </dd>
 
-Different [=users=] will want to share different kinds and amounts of
-[=ancillary data=] with [=sites=]. Some [=people=] will not want to share any
-[=ancillary data=] at all.
+  <dt><dfn>Summarizing ancillary APIs</dfn></dt>
+  <dd>
+    APIs that filter or summarize information available from [=non-ancillary
+    APIs=], like the [[[event-timing]]] and <a
+    data-cite="intersection-observer#introduction">IntersectionObserver</a>.
+  </dd>
 
-Users may be willing to share [=ancillary data=] if it is aggregated with
-the data of other users, or [=de-identified=]. This can be useful
-when [=ancillary data=] contributes to a collective benefit in a way
-that reduces privacy threats to individuals (see <a href="#principle-collective-privacy">collective
-privacy</a>).
+  <dt><dfn>Novel ancillary APIs</dfn></dt>
```

Comment:
Maybe "new sources of ancillary data"?

Reply:
I could do that or 

```suggestion
  <dt><dfn>Ancillary APIs that provide new information</dfn></dt>
```

along with `<dfn>Ancillary APIs that filter or summarize other sources of information</dfn>` above to stay parallel. I'll give other folks a chance to weigh in before making this change.

Reply:
@npdoty suggested using the ["unavoidable information exposure"](https://w3ctag.github.io/privacy-principles/#unavoidable-information-exposure) phrase, and I'll try to work that in.